modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -5,6 +5,22 @@
 # to Google Nearline, etc.
 set -o pipefail
 
+# Quiet mode is off by default (show pv output)
+QUIET=false
+
+# If -q is provided, turn on quiet mode
+while getopts "q" opt; do
+    case $opt in
+        q)
+            QUIET=true
+            ;;
+        \?)
+            # Invalid option, just exit
+            exit 1
+            ;;
+    esac
+done
```

Comment:
meh. parsing command line options in bash is the worst.

can you just check if the shell is interactive? or just make QUIET an environment variable